apex-app-archetype/src/main/resources/archetype-resources/src/main/resources/META-INF/properties.xml

```diff
@@ -13,6 +13,10 @@
   </property>
   -->
   <property>
+    <name>dt.attr.CONTAINER_JVM_OPTIONS</name>
+    <value>-Dlog4j.configuration=log4j.props</value>
+  </property>
+  <property>
```

Comment:
right, changing to log4j.properties. Need to make classpath changes to use name log4j.properties